Gives CMAKE_DOC_DIR undefined, this is half-baked cmake
so don't truly need
---
diff --git Source/cmSystemTools.cxx Source/cmSystemTools.cxx
index 747cedc..7c516d8 100644
--- Source/cmSystemTools.cxx
+++ Source/cmSystemTools.cxx
@@ -2171,10 +2171,6 @@ void cmSystemTools::FindCMakeResources(const char* argv0)
     std::string const prefix =
       exe_dir.substr(0, exe_dir.size() - cmStrLen(CMAKE_BIN_DIR));
     cmSystemToolsCMakeRoot = cmStrCat(prefix, CMAKE_DATA_DIR);
-    if (cmSystemTools::FileExists(
-          cmStrCat(prefix, CMAKE_DOC_DIR "/html/index.html"))) {
-      cmSystemToolsHTMLDoc = cmStrCat(prefix, CMAKE_DOC_DIR "/html");
-    }
   }
   if (cmSystemToolsCMakeRoot.empty() ||
       !cmSystemTools::FileExists(
